scripts/typescript_checks.py

```diff
@@ -27,8 +27,759 @@
 
 from typing import Optional, Sequence
 
+import yaml
+
 from . import common
 
+# Contains the name of all files that is not strict typed.
+# This list must be kept up-to-date; the changes (only remove) should be done
+# manually.
+# Please keep the list in alphabetical order.
+# NOTE TO DEVELOPERS: do not add any new files to this list without asking
+# @vojtechjelinek first.
+NOT_FULLY_TYPE_STRICT_TSCONFIG_FILEPATH = [
+    'core/templates/App.ts',
+    'core/templates/AppSpec.ts',
+    'core/templates/Polyfills.ts',
+    'core/templates/app.constants.ajs.ts',
+    'core/templates/components/angular-html-bind/angular-html-bind-wrapper.directive.spec.ts', # pylint: disable=line-too-long
```

Comment:
Use `# pylint: disable=line-too-long, single-line-pragma` and `# pylint: enable=line-too-long, single-line-pragma` at start and end of list instead of adding disable rule at each line individually.

Reply:
Done